app/controllers/concerns/articles_controller.rb

```diff
@@ -30,19 +30,24 @@ def edit
   def update
     @article = Article.find(params[:id])
   if @article.update(article_params)
-   flash[:success] = "Article was updated"
-   redirect_to article_path(@article)
+    flash[:success] = "Article was updated"
+    redirect_to article_path(@article)
   else
-   flash[:success] = "Article was not updated"
-   render 'edit'
+    flash[:success] = "Article was not updated"
+    render 'edit'
   end
   end
 
   def destroy
```

Comment:
Use [ActiveRecord association to find](https://apidock.com/rails/ActiveRecord/Associations/CollectionProxy/find) user's article.

```ruby
def destroy
  @article = current_user.articles.find_by(id: params[:id])
  if @article.present?
    @article.destroy
    flash[:success] = "Article was deleted"
    redirect_to articles_path
  else
    flash[:danger] = "You are not authorised to perform this action."
    redirect_to articles_path
  end
end

```

Reply:
also fix the indentations in this whole file.